docs/Flint.schema.json

```diff
@@ -0,0 +1,681 @@
+{
+    "$schema": "https://json-schema.org/draft/2019-09/schema#",
+    "type": "object",
+    "definitions": {
+        "source": {
+            "type": "object",
+            "properties": {
+                "validFrom": {
+                    "type": "string",
+                    "pattern": "^$|^[0-9]{2}-[0-9]{2}-[0-9]{4}$"
+                },
+                "validTo": {
+                    "type": "string",
+                    "pattern": "^$|^[0-9]{2}-[0-9]{2}-[0-9]{4}$"
+                },
+                "citation": {
+                    "type": "string"
+                },
+                "juriconnect": {
+                    "type": "string"
+                },
+                "text": {
+                    "type": "string"
+                },
+                "explanation": {
```

Comment:
Shouldn't https://github.com/discipl/flinteditor-mps/pull/190 change the explanation property to comment as an array of strings?

Reply:
Changes specific to #190 should be done in that PR. This is based on master.

As far as i understood #190 won't be merged into master because we stopped the development phase and entered documentation phase. #190 also changes import/export/models/generation warnings and has some conflicts with master.

Reply:
@cblazo please resolve the conversation if the answer is satisfactory

Reply:
Ah I see, thought it would still be pushed to master. In that case I will assume what's on master indeed.

Reply:
@TristanAlbers I am not able to resolve comment threads, I think.